### PR TITLE
Add admin panel endpoints

### PR DESCRIPTION
**Pull request by KalitaKseniya**

### CODE REVIEW

File: BC.AuthenticationMicroservice/Boundary/Request/RegisterRequest.cs [hunk 1]
```diff
@@ -21,5 +21,8 @@ public class RegisterRequest
         [Compare("Password")]
         [DataType(DataType.Password)]
         public string PasswordConfirm { get; set; }
+
+        [Required]
+        public string Role { get; set; }
```
Comment (Sputnavi) [inline BC.AuthenticationMicroservice/Boundary/Request/RegisterRequest.cs:26]: The user shouldn't be able to set a role for himself

File: BC.AuthenticationMicroservice/Models/UserRole.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿using Microsoft.AspNetCore.Identity;
+
+namespace BC.AuthenticationMicroservice.Models
+{
+    public class UserRole : IdentityUserRole<string>
```
Comment (Sputnavi) [inline BC.AuthenticationMicroservice/Models/UserRole.cs:5]: Why are we extending the embedded solution?

File: BC.AuthenticationMicroservice/Boundary/Request/RegisterRequest.cs (hunk 1 above)
Comment (KalitaKseniya) [reply BC.AuthenticationMicroservice/Boundary/Request/RegisterRequest.cs:26]: I assumed to use in the endpoint (UsersController/CreateUser) in admin panel, i.e. for admin to create another admin/user/master, not for user registration. But we also need model for user registration (as you did it in Authentication/RegisterAsync). I can create two models: one for your endpoint (for user registration, without role) and another for admin panel (with required role). What do you think?

File: BC.AuthenticationMicroservice/Models/UserRole.cs (hunk 2 above)
Comment (KalitaKseniya) [reply BC.AuthenticationMicroservice/Models/UserRole.cs:5]: To get users with roles (via Include) in one query. For example, you can see it in UsersService in a method "GetUsersWithRoles".

**Pull request merged**